app/controllers/cart_controller.rb

```diff
@@ -0,0 +1,29 @@
+class CartController < ApplicationController
+  before_action :init_cart, only: %i[update]
+  
+  def show
+    @cart = Cart::CartManager.new(session, params)
+  end
+
+  def update
+    service = Cart::CartManager.new(session, params).call
```

Comment:
свій же код не знаєте
```suggestion
    service = Cart::CartManager.call(session, params)
```